exercises/concept/lasagna-master/lasagna-master.go

```diff
@@ -0,0 +1,11 @@
+package lasagna
+
+/**
+ * Implement the functions needed to solve the exercise here.
+ * Here an example of the syntax as reminder:
+ *
+ * function yourFunction(param string, otherParam int) string {
+ *   ...
+ *   return "foo"
+ * }
+ */
```

Comment:
I'd recommend doing the same here as in the [lasagna-exercise](https://github.com/exercism/go/blob/main/exercises/concept/lasagna/lasagna.go#L5), where there are TODOs for individual functions.

```suggestion
// TODO: define the 'CookingStatus()' function
```